tests/contrib/test_discord.py

```diff
@@ -39,6 +39,24 @@ def test_blueprint_factory():
     assert discord_bp.token_url == "https://discord.com/api/oauth2/token"
 
```

Comment:
You should add an `assert` to `test_blueprint_factory()` too, to ensure the default prompt value of consent is now there.